src/Command/PingStaleIssuesCommand.php

```diff
@@ -61,7 +61,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return 1;
         }
 
-        $notUpdatedAfter = new \DateTimeImmutable(self::STALE_IF_NOT_UPDATED_SINCE);
+        /** @var string $timeString */
+        $timeString = $input->getOption('not-updated-for');
+        $notUpdatedAfter = new \DateTimeImmutable($timeString);
```

Comment:
```suggestion
        $notUpdatedAfter = new \DateTimeImmutable('-'.ltrim($timeString, '-'));
```

I believe this has to be negative (given the previous const value), but the option is as a positive value.

Reply:
That is true. I forgot about that. 

You cannot have an argument or option value starting with a dash.. the console component will correctly intrepid that as a new option. 

I'll update the PR.